CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### Unreleased
 
+## v2.0.0 (2024-09-27)
+
+* Support PHP 8.3
```

Comment:
Maybe just mention the testsupport /excluding other testcases from the package (since that's the BC break)?